### PR TITLE
Email templates migration

### PR DESCRIPTION
**Pull request by EvanBarbour3**
## Features
- Added `Template` model and created pivot tables for Templates

[Delivers [#120757529](https://www.pivotaltracker.com/story/show/120757529)]

### CODE REVIEW

File: app/Basket/Template.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+ * Template Model
+ *
+ * @author EB
+ * @property int    $id
+ * @property int    $merchant_id
+ * @property string $html
+ * @property string $name
+ * @property        $created_at
+ * @property        $updated_at
+ * @property Merchant $merchant
```
Comment (wnowicki) [inline app/Basket/Template.php]: - CS
- CS

File: app/Basket/Template.php [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        'merchant_id',
+        'html',
+    ];
+
+    /**
+     * Date attributes
+     *
+     * @var array
+     */
+    protected $dates = [
```
Comment (wnowicki) [inline app/Basket/Template.php]: ??

**Comment by wnowicki** [comment]
@EvanBarbour3

**Comment by EvanBarbour3** [comment]
@wnowicki :)

**Comment by EvanBarbour3** [comment]
@wnowicki

**Comment by wnowicki** [comment]
waiting for checks

**Pull request merged**